employee&managment/db.js

```diff
@@ -0,0 +1,21 @@
+// db.js
+const { Sequelize } = require('sequelize');
+const { DB_USERNAME, DB_PASSWORD, DB_HOST, DB_PORT, DB_NAME } = require('./config');
+
+const sequelize = new Sequelize(DB_NAME, DB_USERNAME, DB_PASSWORD, {
+  host: DB_HOST,
+  port: DB_PORT,
+  dialect: 'mariadb',
+  logging: console.log,
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Review database logging configuration</b></div>
<div id="fix">

The `logging` option is set to `console.log`, which might lead to excessive logging in production environments. Consider using a more configurable logging approach or disabling it for production.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -9,1 +9,1 @@
 -  logging: console.log,
 +  logging: process.env.NODE_ENV === 'production' ? false : console.log,
```
</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/rishabhbohra3/Bito-CRA-Issues-Dataset/pull/3#issuecomment-2510872065>#7d3d92</a></i></small>
</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged